File: TsogosunProfileAdmin/MSProfileAdmin/Model/Dtos/UserApplicationSectionDto.cs

using System.Collections.Generic;

namespace tsogosun.com.MSProfileAdmin.Model.Dtos
{
    public class UserApplicationSectionDto : UserRightsSectionDto
    {
        public UserDto UserDto { get; set; }
        public List<ApplicationDto> Applications { get; set; }
    }
}

File: TsogosunProfileAdminWeb/src/app/model/Dtos/profile-admin/userApplicationSectionDto.ts
import { ApplicationDto } from "../../loginDto/applicationDto";
import { UserRightsSectionDto } from "../../loginDto/user-rights-sectionDto";
import { UserDto } from "./userDto";

export interface UserApplicationSectionDto extends UserRightsSectionDto {
    userDto?: UserDto | undefined;
    applications?: ApplicationDto[] | undefined;
}